Timebox2/bigwatson/logic/censor_manager.py

```diff
@@ -55,5 +54,73 @@ def censor_article(article, good_class):
             sentences[i] = '<del>' + sentence + '</del>'
 
     censored_body = '. '.join(sentences)
-    censored.body = censored_body
+    return censored_body
+
+
+def censor_title_and_summary(article, good_class):
+    """ censors and returns an article's title and summary. """
+
+    print(good_class)
+    POSSIBLE_CLASSES = ['positive', 'negative']
+
+    censored = Article.Article.from_article(article)
+
+    summary_sents = censored.summary.split('.')
+    for i in range(len(summary_sents)):
+        sentence = summary_sents[i]
+        cls = ''
+        conf = -1
+
+        #fragments from splitting on periods
+        if len(sentence) > 0:
+            cls, conf = classify_sentence(sentence)
+            print(cls, conf)
+
+        #basically if nuetral and very confident
+        if good_class not in POSSIBLE_CLASSES:
+            if conf > .9:
+                try:
+                    print("Strong Feelings: " + sentence)
+                except UnicodeEncodeError:
+                    print("unicode shit")
+                summary_sents[i] = '<del>' + sentence + '</del>'
+        #if not what we're aiming for
+        elif cls != good_class and conf > .4:
+            try:
+                print("Bad hombre:" + sentence)
+            except UnicodeEncodeError:
+                print("unicode shit")
+            summary_sents[i] = '<del>' + sentence + '</del>'
+
+    title_sents = censored.title.split('.')
+    for i in range(len(title_sents)):
```

Comment:
Just fyi, you can just do `for sentence in title_sents:`

Would probably be a bit easier to read.